packages/core/src/__tests__/__snapshots__/caip10-link.test.ts.snap

```diff
@@ -264,6 +264,7 @@ Object {
     },
   ],
   "metadata": Object {
+    "controller": "0x8fe2c4516e920425e177658aaac451ca0463ed69@eip155:1337",
```

Comment:
Why is this exposed on existing stream types? I thought we wanted only to do it on new ones. Seems like this adds a bunch of duplication?

Reply:
I thought it would be good to have the `controller` field show up in all streamtypes, so applications that use multiple streamtypes (like cerscan for example) doesn't have to look for the controller in a different place depending on streamtype.  The 2 new streamtypes now only have `controller`, but the old streamtypes have both `controller` and `controllers` - `controllers` for backwards compatibility, `controller` for consistency with other streamtypes.

If you'd prefer, I could leave TileDoc and Caip10Link with only having `controllers` and have Model and MID only have `controller`